test/extended/build_context_dir_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration,extended,!no-etcd,!no-docker,docker
+
+package extended
+
+import (
+	"testing"
+	"time"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/test/integration"
```

Comment:
Instead of having those bits and pieces here and there, let's create a `test/util` package and put there all the utility stuff such as `RequireDocker`, `RequireEtcd`, etc.